src/v0/destinations/june/data/JuneGroupConfig.json

```diff
@@ -28,7 +28,23 @@
     "sourceFromGenericMap": true,
     "required": false,
     "metadata": {
-      "excludes": ["groupId"]
+      "excludes": [
+        "groupId"
+      ]
+    }
+  },
+  {
+    "destKey": "context",
+    "sourceKeys": "context",
+    "required": false,
+    "metadata": {
+      "excludes": [
+        "page",
+        "traits",
+        "sessionId",
+        "sessionStart",
+        "consentManagement"
+      ]
     }
```

Comment:
@mihir-4116 @aashishmalik seems like some issue with back-merge PR - review the June/data config changes & raise PR to release branch to fix these files

Reply:
Removed duplicate context from June Group Config